2025/incoming/all-i-want-for-christmas-is-the-right-aspect-ratio.pod

```diff
@@ -0,0 +1,459 @@
+Author: Dave Cross <dave@perlhacks.com>
+Title: All I Want for Christmas Is the Right Aspect Ratio
+Topic: App::BlurFill
+
+=encoding utf8
+
+=head2 A bad start to Christmas
+
+=for :html
+<img src="pixie-the-elf_blur.png">
+
+Pixie the Elf (whose parents had strange ideas about names) was not having
+a good December.
```

Comment:
Somewhere there's a joke about Pixie and Pixel that the "strange idea" either overpowers or subverts.

Reply:
Maybe next year :-)